scripts/gitlab/test_linux_stable.sh

```diff
@@ -4,7 +4,7 @@ set -e
 #shellcheck source=../common/lib.sh
 source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/../common/lib.sh"
 
-time cargo test --all --release --verbose --locked --features=runtime-benchmarks --features=real-overseer
+time cargo test --all --release --verbose --locked --features=runtime-benchmarks
 
 cd parachain/test-parachains/adder/collator/
-time cargo test --release --verbose --locked --features=real-overseer
+time cargo test --release --verbose --locked
```

Comment:
I believe we don't need it anymore, should be covered by `cargo test --all` above